openshift-storage-libs/openshiftstoragelibs/baseclass.py

```diff
@@ -581,6 +589,155 @@ def create_heketi_volume_with_name_and_wait(
 
         return h_volume_info
 
+    def cleanup_heketi_blockhosting_volumes(
+            self, bhv_initial, skip_cleanup_bv=False, raise_on_error=False):
+        """Delete heketi blockhosting volumes and blockvolumes.
+
+        Args:
+            initial_vols (list): Volumes present at initial
+            skip_cleanup_bv (bool): skip cleanup or not.
+        """
+        bhv_final = list(
+            get_block_hosting_volume_list(
+                self.heketi_client_node, self.heketi_server_url).keys())
+
+        diff_volumes = list(set(bhv_final) - set(bhv_initial))
+        for volume in diff_volumes:
+            h_vol_info = heketi_volume_info(
+                self.heketi_client_node, self.heketi_server_url,
+                volume, json=True)
+            if not skip_cleanup_bv and h_vol_info.get("block", {}):
+                for block_vol in (
+                        h_vol_info.get(
+                            "blockinfo", {}).get("blockvolume", {})):
+                    heketi_blockvolume_delete(
+                        self.heketi_client_node, self.heketi_server_url,
+                        block_vol)
+            heketi_volume_delete(
+                self.heketi_client_node, self.heketi_server_url,
+                volume, raise_on_error=raise_on_error)
+
+    def create_heketi_volumes_in_batch(
+            self, vol_count, size=1, batch_amount=8, timeout=300, interval=5,
```

Comment:
Better to change `size` to `vol_size` and `batch_amount` to `batch_count`

Reply:
These variable are used in the library, so keeping the same viable name